src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,20 +86,44 @@
 
             TEAMMATES_VERSION = extractVersionNumber(FileHelper.readFile("src/main/webapp/WEB-INF/appengine-web.xml"));
 
-            TEST_ADMIN_ACCOUNT = prop.getProperty("test.admin.account");
-            TEST_ADMIN_PASSWORD = prop.getProperty("test.admin.password");
+            if (isCiEnvironment() || isGodModeEnabled()) {
```

Comment:
You're right that dev server does not need the password at all, so setting them to null should not be a problem but I might be wrong. Just to be defensive, let's add a condition `isDevServer()` here so the condition will read as `isDevServer() && (isCiEnvironment() || isGodModeEnabled())`.